LayoutTests/TestExpectations

```diff
@@ -7740,3 +7740,11 @@ imported/w3c/web-platform-tests/css/CSS2/floats/float-nowrap-hyphen-rewind-1.htm
 
 # This test is flaky. Resizing the window exposures the failure
 webkit.org/b/288689 svg/text/bidi-text-query.svg [ Skip ]
+
+# web-platform-tests/css/css-ui/compute-kind-widget-generated failures
```

Comment:
Can you add to the following section instead of creating a separate one ?https://searchfox.org/wubkat/rev/662cdccd8338a60684dac2fc6e14d26305e5db8a/LayoutTests/TestExpectations#6135

Reply:
Fixed - Sorry I missed this list.